src/WordPressImportBundle/Service/Importer.php

```diff
@@ -208,11 +208,6 @@ protected function importNews(Client $client, $objPost, NewsArchiveModel $objArc
             return;
         }
 
-        // check if post already exists (skip)
-        if (NewsModel::countBy(['wpPostId = ?', 'pid = ?'], [$objPost->id, $objArchive->id]) > 0) {
-            return;
-        }
-
```

Comment:
Ah, now I remember. Yes, the check is also done in `Importer::getPostsFromWordpress`, but I think I have added the check here again just for safety - in case `Importer::import` is called somewhere else in the future.

Reply:
I think you should check the code a bit closer ;) This is in a protected method you can't call. `Importer::import()` calls `getPostsFromWordpress()` which does the check :)

Reply:
Yes, well I meant within `Importer`, but it's fine anyway ;)